packages/portal/frontend/src/app/views/AdminConfigTab.ts

```diff
@@ -559,6 +561,16 @@ export class AdminConfigTab extends AdminPage {
 
     private async updateClasslistPressed(): Promise<void> {
         Log.trace('AdminConfigTab::updateClasslistPressed(..) - start');
+
+        const mapToTextAndSubtext = function(people: Person[]) {
+            return people.map(function(person) {
+                return {
+                    text: person.studentNumber + '/' + person.githubId + ': ' + person.fName + ' ' + person.lName,
```

Comment:
I'd also add `person.id` to this if it is different than `githubId` (Trying to have both throughout the interface just so we can make it easier to get at whatever format the instructional staff is using internally)

Reply:
I added `person.id`, as it will be different than `githubId`. 

If there is any additional feedback on the UI, we can modify it to better suit people's needs. Some ideas:
- exporting this information to CSV/JSON.
- withdraw button option at bottom of removal dialog box.